coderd/database/queries/workspaceresources.sql

```diff
@@ -47,11 +47,14 @@ FROM
 WHERE
 	workspace_resource_id = ANY(@ids :: uuid [ ]);
 
--- name: InsertWorkspaceResourceMetadata :one
+-- name: InsertWorkspaceResourceMetadata :many
```

Comment:
I think the selects above should have explicit `id ASC`s. I think that'd be the default, but changing the primary key again could alter that.